docs/sql-programming-guide.md

```diff
@@ -1640,6 +1640,129 @@ Configuration of Hive is done by placing your `hive-site.xml`, `core-site.xml` a
 You may run `./bin/spark-sql --help` for a complete list of all available
 options.
 
+# PySpark Usage Guide for Pandas with Arrow
+
+## Arrow in Spark
```

Comment:
let's say "Apache Arrow" here and in L1643? (ASF policy...)

Reply:
Wow, I just learnt it .. 👍

Reply:
Maybe we should also add "Apache" to the headline above?

Reply:
yes, what I mean by L1643.
We should at least use the full name in header(s) and the first mention before simplifying it to "Arrow" afterwards